integration_tests/features/steps/fal_steps.py

```diff
@@ -1,16 +1,28 @@
 from functools import reduce
 import os
-from typing import Dict
+from typing import Dict, List
 from behave import *
 from fal.cli import cli
 import tempfile
 import json
 import unittest
 from os.path import exists
+from enum import Enum
 from pathlib import Path
+from datetime import datetime, timezone
 import re
 
 
+class FalArtifact(int, Enum):
+    # The main distinction we can use on an artifact file to determine
+    # whether it was created by a Python script or a Python model is the number
+    # of suffixes it has. Models use <model_name>.txt and scripts use
+    # <model_name>.<script_name>.txt
+
+    MODEL = 1
+    SCRIPT = 2
```

Comment:
I know we are just doing tests... But this feels hacky for me a little bit. Enum values are not normally used to check against like we are doing for the artifacts' `len(artifact.suffices) in kinds`?

Reply:
It actually doesn't have to be an int enum, just thought it would be better to encapsulate in a different namespace (`FalArtifact`) though no strong opinions. WDYT about the current version (just two top-level constants, `FAL_MODELS` and `FAL_SCRIPTS`)?